libr/core/cmd_anal.c

```diff
@@ -1460,6 +1460,10 @@ static int var_cmd(RCore *core, const char *str) {
 			var_help (core, type);
 			break;
 		}
+		if (!fcn) {
```

Comment:
maybe worth using r_str_trim apis instead of this for+strchr, also the for loop is not using braces {}